doc/userguide.rst

```diff
@@ -597,6 +597,39 @@ Known limitations
 This section lists those limitations that are unlikely to change, as they come
 from external sources.
 
+Restarting the JVM
+~~~~~~~~~~~~~~~~~~
+
+JPype caches many data structures within the JVM. Those resource are still
+allocated after the JVM is shutdown as there are still Python objects 
+that point to those resources.  If the JVM is restarted, those stale Python
+objects will be in a broken state and the fresh startiJVM will reallocated the
```

Comment:
and the new JVM instance will obtain the references to these resulting in a memory leak.